### ISSUE TITLE
Trying to adapt to a new dataset and error occurs.

### ISSUE THREAD

**Issue by ElliottYan** [TimDettmers/ConvE#50]
![image](https://user-images.githubusercontent.com/10862038/63571431-c024d280-c5b2-11e9-805a-65c195cffad5.png)

Hi, I'm trying to train ConvE in NELL-995 dataset, but there's an error occurs. Do you have any idea how to solve this? 

Thanks.

**Comment by TimDettmers** [comment]
It seems this error is related to not finding the next batch. It might have to do that your datasets has a small validation set and you run with a batch size which is too large or with too many `loader_threads`.

I would the following things in this order: Delete the preprocessed data and try again. If that does not work try to change the batch size (smaller). If that does not help try to reduce the number of `loader_threads`. After that, I am not sure what it could be and I need a bit more information about your dataset such as the number of triples in train/test/validation set.

**Comment by ElliottYan** [comment]
I try to change the number of *loader_threads* and the program now seems fine to me. Thanks for your help. 
I'm closing this issue.
